AirCasting/CoreData/Storages/UIStorage.swift

```diff
@@ -9,8 +9,8 @@ protocol UIStorage {
 }
 
 protocol UIStorageContextUpdate {
-    func cardStateToggle(for sessionUUID: SessionUUID) throws
-    func changeStream(for sessionUUID: SessionUUID, stream: String) throws
+    func cardStateToggle(for sessionUUID: SessionUUID, isSessionExternal: Bool) throws
```

Comment:
I don't like that we need to pass `sessoinExternal` in here. It should be handled internally IMO. `SessionUUID` should be unique across all sessions